src/In.ProjectEKA.HipService/DataFlow/Model/DataFlowRequest.cs

```diff
@@ -0,0 +1,21 @@
+namespace In.ProjectEKA.HipService.DataFlow.Model
+{
+    using System.ComponentModel.DataAnnotations;
+
+    public class DataFlowRequest
+    {
+        [Key] public string TransactionId { get; }
+        
+        public HealthInformationRequest HealthInformationRequest { get; }
```

Comment:
HealthInformationRequest itself has a transactionID, why one more transactionID?

Reply:
It is acting as a key